pkgs/applications/misc/citations/default.nix

```diff
@@ -62,6 +60,13 @@ stdenv.mkDerivation (finalAttrs: {
     darwin.apple_sdk.frameworks.Foundation
   ];
 
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang (lib.concatStringsSep " " [
```

Comment:
this works, but maybe @wegank can suggest a better fix for darwin

Reply:
This looks good enough to me. Probably @reckenrode can suggest a better fix.